hack/tools/vault_utils.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+: ${PORT_TO_FORWARD:=8200}
+
+# $1 role name
+# $2 policy name
+# $3 path to auth method
+# $4 bound_service_account_namespaces
+create_role() {
```

Comment:
This role creation will be moved to the application controller at some point when every module will be deployed with it's own service account right?

Reply:
Right.